tests/compute/empty-struct2.slang

```diff
@@ -0,0 +1,43 @@
+//TEST(smoke,compute):COMPARE_COMPUTE:
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):dxbinding(0),glbinding(0),out
+// This is a basic test for Slang compute shader.
+
+RWStructuredBuffer<float> outputBuffer;
```

Comment:
In general I find that these end-to-end compute tests are easier to create when the values we store out to memory are `int`s or `uint`s, and we've disabled most of the sugar in the test runner for using floating-point syntax for the expected output. Is there a reason you are preferring `float` here?

It really doesn't matter, except that people will have a tendency to generate the expected output by copying over the actual output once they have things working, instead of working through what they expect to see in the output by hand (e.g., I try whenever possible to hand-author the expected output based on my understanding of what should happen, so that I can avoid tests that "work," but not like I expect).

Reply:
Changed to uint.